### PR TITLE
New ChestMenu option

### PR DESCRIPTION
**Pull request by VoidAngel**
Mostly just to remove the annoying metrics spam D:

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
What spam?

**Comment by VoidAngel** [comment]
Ah sorry just realized the source of the spam was a 3rd party plugin pointing at CS-CoreLib. 
I initially thought it was just because mcstats is dead.

**Comment by TheBusyBiscuit** [comment]
Alright, but as far as I know it isn't dead, just unavailable for most of the time.
However I also heard that the domain expires this month, so a switch may be an option.

I will look into bstats.

File: src/org/bstats/Metrics.java [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+ * Check out https://bStats.org/ to learn more about bStats!
+ */
+public class Metrics {
+
+    static {
+        // You can use the property to disable the check in your test environment
+        if (System.getProperty("bstats.relocatecheck") == null || !System.getProperty("bstats.relocatecheck").equals("false")) {
+            // Maven's Relocate is clever and changes strings, too. So we have to use this little "trick" ... :D
+            final String defaultPackage = new String(
+                    new byte[]{'o', 'r', 'g', '.', 'b', 's', 't', 'a', 't', 's', '.', 'b', 'u', 'k', 'k', 'i', 't'});
```
Comment (TheBusyBiscuit) [inline src/org/bstats/Metrics.java]: It throws an exception when not placed in "org.bstats.bukkit"
It isn't placed there.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/org/bstats/Metrics.java (hunk 1 above)
Comment (VoidAngel) [reply src/org/bstats/Metrics.java]: Are you sure? I was under the impression that it only threw the exception if the package string was equal to "org.bstats.bukkit" or "your.package"
`            if (Metrics.class.getPackage().getName().equals(defaultPackage) || Metrics.class.getPackage().getName().equals(examplePackage)) {
                throw new IllegalStateException("bStats Metrics class has not been relocated correctly!");
            }`

**Comment by VoidAngel** [comment]
Anywayssss no pressure xD Now that I see this isn't the source of my metrics spam, I know this doesn't really affect server owners, so I'll leave it up to you

File: src/org/bstats/Metrics.java (hunk 1 above)
Comment (TheBusyBiscuit) [reply src/org/bstats/Metrics.java]: Aaaaaand that is why I shouldn't be at my pc at 10 am.

**Review by TheBusyBiscuit — DISMISSED** [review]

**Comment by VoidAngel** [comment]
PS I can remove the bstats thing if you want.

**Comment by TheBusyBiscuit** [comment]
Your ChestMenu implementation looks good, exactly what I meant.

I haven't made up my mind on bstats yet but I would prefer to seperate these two by moving bstats on a different branch and making a Pull Request that can sit around till I made a decision.

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**